src/test/java/org/traccar/protocol/Mavlink2ProtocolDecoderTest.java

```diff
@@ -0,0 +1,16 @@
+package org.traccar.protocol;
+
+import org.junit.Test;
+import org.traccar.ProtocolTest;
+
+public class Mavlink2ProtocolDecoderTest extends ProtocolTest {
+
+    @Test
+    public void testDecode() throws Exception {
+        var decoder = new Mavlink2ProtocolDecoder(null);
+        verifyAttributes(decoder, binary("fd1c00006b010121000068a705004005d323b89aa30e1cfdffff17fdffffa4fffdff06001c2d0000"));
```

Comment:
Are there any significant differences between the 4 samples? If not, just one or two should be more than enough.